modules/nw-ingress-controller-configuration-gcp-global-access.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-gcp-global-access_{context}"]
+= Configuring Global Access parameters on GCP
+As a cluster administrator with an internal load balancer on GCP, you can specify the global access option. Clients in any region within the same VPC network as the load balancer can reach the workloads running on your cluster.
+
+For more information, please reference the GCP documentation for link:https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access[Global access].
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on GCP infrastructure.
+* You created an ingress controller with internal load balancing.
+* You installed the `oc` command line interface.
+
+.Procedure
+
+. Configure the 'clientAccess' field by specifying the appropriate level for 'clientAccess', global and local.
++
+.. Edit the Ingress Controller Operator:
```

Comment:
This field is set on the IngressController resource, not the cluster Ingress config. So this section of this PR needs to be revised accordingly.